src/dialogs/more-info/components/lights/light-color-rgb-picker.ts

```diff
@@ -115,46 +115,50 @@ class LightRgbColorPicker extends LitElement {
       </div>
       ${supportsRgbw || supportsRgbww
         ? html`<ha-labeled-slider
+            labeled
             .caption=${this.hass.localize("ui.card.light.color_brightness")}
             icon="hass:brightness-7"
+            min="0"
             max="100"
             .value=${this._colorBrightnessSliderValue}
-            @change=${this._colorBrightnessSliderChanged}
-            pin
+            @value-changed=${this._colorBrightnessSliderChanged}
```

Comment:
If we are using the `value-changed` event, we should also use the value of the event, instead of using `ev.target.value`